### PR TITLE
<Input /> - story according new guidelines

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6795/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6795/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6795

File: packages/wix-style-react/src/Input/docs/examples.js [hunk 1]
```diff
@@ -1,248 +1,159 @@
...
-  </Cell>
-  <Cell>
-    <Input
-      prefix={<Input.Affix>https://</Input.Affix>}
-      suffix={<Input.Affix>.com</Input.Affix>}
-    />
-  </Cell>
-  <Cell>
+export const clearButton = `
+() => {
```
Comment (mykas) [inline packages/wix-style-react/src/Input/docs/examples.js:67]: Lets minimise this example to:
```
() => {
  const [inputText, setInputText] = React.useState(
    'Click clear button to erase this value',
  );

  return (
    <Input
      value={inputText}
      clearButton
      onChange={e => setInputText(e.target.value)}
      onClear={() => setInputText('')}
    />
  );
};
```

File: packages/wix-style-react/src/Input/docs/examples.js [hunk 2]
```diff
@@ -1,248 +1,159 @@
...
+    </Cell>
+    <Cell span={10}>
+      <Input placeholder="00 000-0000" />
+    </Cell>
+  </Layout>
+</FormField>;
 `;
 
-export const border = `
+export const inputAsTitle = `
```
Comment (mykas) [inline packages/wix-style-react/src/Input/docs/examples.js:114]: I think this example is a bit too complicated to understand. Lets follow this guideline tip to make it more readable https://gist.github.com/mykas/6fa3cac9e08a5c6455ef2ce5545739e1#4-complex-structure-components-examples-can-be-daunting-to-comprehend-in-such-cases-there-is-value-in-abstracting-code-into-smaller-modules-in-order-to-reduce-mental-effort-in-understanding-the-code

File: packages/wix-style-react/src/Input/docs/examples.js [hunk 3]
```diff
@@ -1,248 +1,159 @@
...
-    />
-  </Cell>
-  <Cell>
-    <Input
-      size="large"
-      placeholder="They did not know it was impossible, so they did it!"
-    />
-  </Cell>
-</Layout>
+export const compoundInput = `
```
Comment (mykas) [inline packages/wix-style-react/src/Input/docs/examples.js:93]: This is snippet is also a bit too much to read. Lets follow these tips to construct better example:
1. Large data sets should be added to meaningfull variables: https://gist.github.com/mykas/6fa3cac9e08a5c6455ef2ce5545739e1#3-in-case-property-requires-large-data-sets-for-interpretation---functional-snippets-can-be-used-to-move-big-data-sets-to-meaningful-variables
2. https://gist.github.com/mykas/6fa3cac9e08a5c6455ef2ce5545739e1#3-in-case-property-requires-large-data-sets-for-interpretation---functional-snippets-can-be-used-to-move-big-data-sets-to-meaningful-variables

File: packages/wix-style-react/src/Input/docs/examples.js (hunk 1 above)
Comment (egidijusja) [reply packages/wix-style-react/src/Input/docs/examples.js:67]: Fixed

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@egidijusj

- [ ] Please make "compound input" example with format validation. Egozi's example can be used: https://github.com/wix/wix-style-react/issues/4627
image
- [ ] Let's change input's value next to image to "Cooking recipes album"
image
- [ ] Please add text formatting to Feedback section:
"You can help us improve this component by providing feedback, asking questions or leaving any other comments via **#wix-style-ux** or **#wix-style-react** Slack channels or GitHub. Found a bug? Please report it to: https://goo.gl/forms/wrVuHnyBrEISXUPF2"
- [ ] Please swap placeholder values:
image

**Comment by egidijusj** [comment]
@milkyfruit you most likely intended to tag @egidijusja, not me

**Review by milkyfruit — APPROVED** [review]
Looks great!

**Review by mykas — APPROVED** [review]

**Pull request merged**